pandas/tools/tests/test_tile.py

```diff
@@ -283,6 +284,16 @@ def test_single_bin(self):
         result = cut(s, 1, labels=False)
         tm.assert_series_equal(result, expected)
 
+    def test_datetime_cut(self):
+        # GH 14714
+        data = to_datetime(Series(['2013-01-01', '2013-01-02', '2013-01-03']))
+        result, bins = cut(data, 3, retbins=True)
```

Comment:
can u also test:

- ``list`` of ``np.datetime64``
- ``ndarray`` of ``datetime64`` dtype
- ``DatetimeIndex``